src/address_family.rs

```diff
@@ -1,19 +1,31 @@
 use net2::UdpBuilder;
+#[cfg(not(windows))]
 use net2::unix::UnixUdpBuilderExt;
 use std::io;
 use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, UdpSocket};
 use super::MDNS_PORT;
+#[cfg(not(windows))]
```

Comment:
this should be
```
#[cfg(target_os = "linux")]
```
as further down the page we only use libc using that condition